### PR TITLE
Modifying home page and post list UI

### PR DESCRIPTION
**Pull request by sswatisharmaa**
Added post creation form and post list to home page, removed comments section, changed post creation date format.

### CODE REVIEW

File: frontend/src/app/components/posts/add-post/add-post.component.ts [hunk 1]
```diff
@@ -19,72 +12,54 @@ export interface Post {
...
+  constructor(private postService: PostService, private route: Router)  { }
 
   ngOnInit(): void {
-
     this.postForm = new FormGroup({
       postBody : new FormControl(),
-      tagField : new FormControl(),
     });
-
     this.postForm.controls.postBody.valueChanges.subscribe(value => {
```
Comment (anurag091) [inline frontend/src/app/components/posts/add-post/add-post.component.ts]: please de-structure it first

File: frontend/src/app/components/posts/add-post/add-post.component.ts [hunk 2]
```diff
@@ -19,72 +12,54 @@ export interface Post {
...
      value === '' ? this.isPostAvailable = false :  this.isPostAvailable = true;
     });
-    this.postForm.controls.tagField.valueChanges.subscribe(value => {
```
Comment (anurag091) [inline frontend/src/app/components/posts/add-post/add-post.component.ts:47]: here again

Comment (sswatisharmaa) [reply frontend/src/app/components/posts/add-post/add-post.component.ts:47]: This code piece has been removed.

File: frontend/src/app/components/posts/add-post/add-post.component.ts (hunk 1 above)
Comment (sswatisharmaa) [reply frontend/src/app/components/posts/add-post/add-post.component.ts]: Kindly validate.

**Review by anurag091 — APPROVED** [review]

**Pull request merged**